src/ngAnimate/animateQueue.js

```diff
@@ -610,65 +610,61 @@ var $$AnimateQueueProvider = ['$animateProvider', /** @this */ function($animate
       activeAnimationsLookup.remove(node);
     }
 
-    function isMatchingElement(nodeOrElmA, nodeOrElmB) {
-      return getDomNode(nodeOrElmA) === getDomNode(nodeOrElmB);
-    }
-
     /**
      * This fn returns false if any of the following is true:
      * a) animations on any parent element are disabled, and animations on the element aren't explicitly allowed
      * b) a parent element has an ongoing structural animation, and animateChildren is false
      * c) the element is not a child of the body
      * d) the element is not a child of the $rootElement
      */
-    function areAnimationsAllowed(element, parentElement, event) {
-      var bodyElement = jqLite($document[0].body);
-      var bodyElementDetected = isMatchingElement(element, bodyElement) || element[0].nodeName === 'HTML';
-      var rootElementDetected = isMatchingElement(element, $rootElement);
+    function areAnimationsAllowed(node, parentNode, event) {
+      var bodyNode = $document[0].body;
+      var rootNode = getDomNode($rootElement);
+
+      var bodyNodeDetected = (node === bodyNode) || node.nodeName === 'HTML';
+      var rootNodeDetected = (node === rootNode);
```

Comment:
A couple of reasons (at least):

1. I find them more readable. I think it is a remnant from my early Java days (and [this ancient document][1]).
2. It is also visually closer to `isMatchingElement(...)` 😛

<sub>(Hey, I changed the trailing dots, I am not removing the parens.)</sub>

[1]: http://www.oracle.com/technetwork/java/codeconventions-150003.pdf